python/pyspark/pandas/internal.py

```diff
@@ -935,10 +935,11 @@ def attach_sequence_column(sdf: PySparkDataFrame, column_name: str) -> PySparkDa
         )
         return sdf.select(sequential_index.alias(column_name), *scols)
 
-    # TODO(SPARK-43610): Enable `InternalFrame.attach_distributed_column` in Spark Connect.
     @staticmethod
     def attach_distributed_column(sdf: PySparkDataFrame, column_name: str) -> PySparkDataFrame:
         scols = [scol_for(sdf, column) for column in sdf.columns]
+        if is_remote():
```

Comment:
Can you add a comment here that we don't add an alias to avoid adding a dedicated protobuf message for cosmetic changes?

Reply:
Just added a comment. Thanks!